rdmd_test.d

```diff
@@ -74,9 +76,18 @@ void main(string[] args)
     enforce(res.status == 0, res.output);
     enforce(rdmdApp.exists);
 
-    runTests(rdmdApp, compiler, model);
-    if (concurrencyTest)
-        runConcurrencyTest(rdmdApp, compiler, model);
+    // if no explicit list of test compilers is set,
+    // use the compiler used to build rdmd
+    if (testCompilerList is null)
+        testCompilerList = compiler;
```

Comment:
Why not just initialize `testCompilerList` to `compiler`? That would make this check unnecessary.

Reply:
Oops, I see now that `compiler` is also retrieved from the command line.